dashboard/app/controllers/abuse_report_controller.rb

```diff
@@ -0,0 +1,43 @@
+require 'json'
+require 'httparty'
+
+class AbuseReportController < ApplicationController
+  def report_abuse
+    HTTParty.post('https://codeorg.zendesk.com/api/v2/tickets.json',
+      headers: {"Content-Type" => "application/json", "Accept" => "application/json"},
+      body: {
+        :ticket => {
```

Comment:
Nit: mixing `key: value` and `:key => value` style hashes.  ([The style guide recommends `key: value`](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-hashes))